target/linux/mpc85xx/image/p1020.mk

```diff
@@ -15,6 +15,17 @@ define Device/aerohive_hiveap-330
 endef
 TARGET_DEVICES += aerohive_hiveap-330
 
+define Device/extreme-networks_ws-ap3825i
```

Comment:
This should go below `enterasys_ws-ap3710i`.
(BTW it seems Enterasys and Extreme Networks are the same vendor, right?)

Reply:
Enterasys (and their designs) were purchased by Extreme Networks during the transition; this AP is identical to the Enterasys WS-AP3715i, but with the ath9k 5GHz card replaced with an ath10k one.

Reply:
Fixed in force-push 557192907b9652b21f1cc8890b807edae22aed40.